doc/user_guide/io.rst

```diff
@@ -167,7 +167,8 @@ default :ref:`hspy-format` format:
 
     >>> s.save('spectrum')
 
-If instead you want to save in the :ref:`ripple-format` write instead:
+If instead you want to save in the :ref:`ripple format <ripple-format>` write
+instead:
```

Comment:
"If you want to save in the :ref:`ripple format <ripple-format>` instead, write:"